### PR TITLE
Add support for Xcode version 5.0.1 and added simple query in text example

### PR DESCRIPTION
**Pull request by flagsoft**
Added support for Version 5.0.1 (5A2053) - can now build for iOS.
Added testSimpleQuery_main() test function with simple query example.

### CODE REVIEW

**Comment by huyz** [comment]
Is this project still alive?

**Comment by flagsoft** [comment]
Hello, it's originally from https://github.com/71squared/TBXML please ask there.

**Comment by huyz** [comment]
This is 71squared/TBXML

**Comment by flagsoft** [comment]
Hello, I don't understand your answer. I am not 71squared. So I am afraid
that I can not answer your questions.
But please let me know if something is wrong. Let's figure out what the
problem is then.

On Fri, Jul 11, 2014 at 10:42 PM, Huy Z notifications@github.com wrote:

> This is 71squared/TBXML
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/71squared/TBXML/pull/24#issuecomment-48780598.

## 

Freundliche Grüsse

Michael Mustun

**Comment by huyz** [comment]
@flagsoft  I'm not emailing you directly.  I'm posting on this "Issue" on  https://github.com/71squared/TBXML .  You do not have to respond :)

This pull request is old but hasn't been accepted so I'm wondering if this project is still alive.  I'm hoping that someone from 71squared/TBXML will respond.

**Comment by balram3429** [comment]
Just remove "self" from the watch list. Or simply do an Unwatch on github.
